.github/actions/maven-build-and-tag/action.yml

```diff
@@ -222,14 +225,47 @@ runs:
         echo "testcontainers.reuse.enable=true" > ~/.testcontainers.properties
         echo "TESTCONTAINERS_RYUK_DISABLED=true" >> $GITHUB_ENV
 
+    - name: Set skip tests env variable
+      id: set-skip-tests-env-variable
+      if: github.event_name == 'pull_request'
+      env:
+        SKIP_TESTS: ${{ contains(github.event.pull_request.labels.*.name, 'skip-tests') }}
```

Comment:
Please take example on the similar "IS_PREVIEW" env variable resolution, so that the variable is set even if the event is not "pull_request"

Reply:
Also please don't hardcode the label name: there are other inputs defined for a label to be specified (it can default to "skip-tests"). Do we want to keep "skip-tests" btw @andrea-ligios or align on "skip-validation"?

Reply:
I don't have strong opinions on that.

Skip Tests sounds more understandable though.